docs/en/changelog.md

```diff
@@ -1,6 +1,50 @@
 ## Changelog
 
 
+### V0.22 (3/04/2022)
+
+**Highlights**
+
+- Officially Support ConvNeXt: A ConvNet for the 2020s. Please use the latest MMClassification (0.21.0) to try it out.
+- Support iSAID aerial Dataset.
+
+**New Features**
+
+- Support Support ConvNeXt: A ConvNet for the 2020s. ([#1216](https://github.com/open-mmlab/mmsegmentation/pull/1216))
+- Support iSAID aerial Dataset. ([#1115](https://github.com/open-mmlab/mmsegmentation/pull/1115)
+- Support kernel updation for some decoder heads (ASPP, FCN, PSP, UPer). ([#1299](https://github.com/open-mmlab/mmsegmentation/pull/1299))
```

Comment:
it might belong to refactor

Split forward function into forward feature and cls_seg?